remme/shared/forms/__init__.py

```diff
@@ -14,6 +14,7 @@
     RevokePubKeyPayloadForm,
 )
 from .account import TransferPayloadForm, GenesisPayloadForm
+from .node_account import NodeAccountInternalTransferPayloadForm, EmptyPayloadForm
```

Comment:
Please, follow the already presented style in the file, so migrate from

```
from .. import .., ..
```

to

```
from .. import (
    .., 
    ..,
)
```

Reply:
Done.